### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by Kingno3l**
- [ ] Used webpack to bundle JavaScript.
- [ ] Used proper ES6 syntax.
- [ ] Used ES6 modules to write modular JavaScript.
- [ ] Created an index.html file and wrote HTML markup. Create an empty To Do List placeholder (<ul> element). The index.html file is set as a template using the HTML Webpack Plugin.
- [ ] Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object contain three keys:
1. description [string].
2. completed [bool].
3. index: [number].

- [ ] Wrote a function to iterate over the tasks array and populate an HTML list item element for each task.
- [ ] On page load rendered the dynamically created list of tasks in the dedicated placeholder. The list appeared in order of the index values for each task.
- [ ] Created a style.css and set rules for the To Do List. CSS is loaded by Webpack Style/CSS Loader. 
- [ ] All source files (index.html, index.js and style.css) are located in /src directory and your distribution files will be generated by webpack and served by webpack dev server from /dist folder.

### CODE REVIEW

File: src/print.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
-export default function printMe() {
-    console.log('I get called from print.js!');
-  }
\ No newline at end of file
+// export default function printMe() {
+//     console.log('I get called from print.js!');
+//   }
```
Comment (zieeco) [inline src/print.js]: - Hey, please, let's remove the commented-out code from our codebase. They make our codebase not clean and maintainable.

File: src/index.js [hunk 2]
```diff
@@ -1,20 +1,91 @@
-import _ from 'lodash';
-import printMe from './print.js';
+// import _ from "lodash";
+// import printMe from "./print.js";
```
Comment (zieeco) [inline src/index.js]: - Hey, please, let's remove the commented-out code from our codebase. They make our codebase not clean and maintainable.

File: src/index.js [hunk 3]
```diff
@@ -1,20 +1,91 @@
...
 import './style.css';
 
- function component() {
-   const element = document.createElement('div');
-  const btn = document.createElement('button');
+// function component() {
+//   const element = document.createElement("div");
+//   const btn = document.createElement("button");
```
Comment (zieeco) [inline src/index.js]: - Hey, please, let's remove the commented-out code from our codebase. They make our codebase not clean and maintainable.

File: src/index.js [hunk 4]
```diff
@@ -1,20 +1,91 @@
...
+//     containerList.innerHTML += `
+//       <li class="list-to-do">
+//         <input type="checkbox" class="to-do">
+//         ${dataToDoList}
+//         <i class="fa-solid fa-trash"></i>
+//       </li>
+//       <hr class="line">
+//     `;
+//   }
+// };
```
Comment (zieeco) [inline src/index.js]: - Hey, please, let's remove the commented-out code from our codebase. They make our codebase not clean and maintainable.

File: src/index.js [hunk 5]
```diff
@@ -1,20 +1,91 @@
...
-   return element;
- }
+//   return element;
+// }
 
- document.body.appendChild(component());
\ No newline at end of file
+// document.body.appendChild(component());
+
+/* eslint-disable no-restricted-syntax */
```
Comment (zieeco) [inline src/index.js]: - Please, kindly remove this eslint disable code here also, as this does not have any effect on your code. Also, it's not ideal to disable the linter checks as this helps you in writing clean and maintainable code to improve your development skill. Let's kindly remove this line.

NOTE: *Please, remove all the commented-out pieces of codes from your codebase to make your codebase clean and maintainable*

File: src/index.js [hunk 6]
```diff
@@ -1,20 +1,91 @@
...
+  const containerList = document.getElementById('list-container');
+  containerList.innerHTML += `
+      <li class="list-to-do">
+        <input type="checkbox" class="to-do">
+        ${output}
+        <i class="fa-solid fa-trash"></i>
+      </li>
+      <hr class="line">
+    `;
+});
```
Comment (zieeco) [inline src/index.js]: - Hey, great job with displaying the task from the todo array, however, you should write a function that would iterate over the tasks array and populate the HTML list item element for each task. Kindly wrap the loop in a function.

File: src/print.js (hunk 1 above)
Comment (zieeco) [reply src/print.js]: ### README FILE


- [ ] Please, kindly do the following to make your readme more professional


   - :point_right: *Update the project's title, Add a description about the project, and remove the placeholder text*
Screenshot from 2023-02-04 11-50-59

---

   - :point_right: *Remove the stack not related to this project and update the Tech Stack to what you used in building this project, and remove the placeholder text*
Screenshot from 2023-02-04 11-57-45


---

   - :point_right: *Add some key features of the project, and remove the placeholder text*
Screenshot from 2023-02-04 12-01-44

---

   - :point_right: *Update the Authors to Author as you've worked on this project alone and update the links to be the links of your social media platforms*
   
Screenshot from 2023-02-04 12-03-54

---

   - :point_right: *Add some future features about the project so users can know what the project would be useful for*
   
Screenshot from 2023-02-04 12-06-12

---

   - :point_right: *Kindly update this content to "If you like this project, give a :star: , and remove the placeholder text "*
Screenshot from 2023-02-04 12-08-21

---

   - :point_right: *Kindly remove this section as it's OPTIONAL, or if you want to add content, then add some question and answers one could ask about the project*
Screenshot from 2023-02-04 12-11-18

---

   - :point_right: *Please, kindly add a license as the link is currently broken and remove the last sentence as they are placeholder text telling how important it is to add a license*
Screenshot from 2023-02-04 12-13-59

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Kingno3l ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 6 above)
Comment (Kingno3l) [reply src/index.js]: Take a closer look.
It's done.
Thank you.

**Review by binhussen — APPROVED** [review]
Hi @Kingno3l , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**